drivers/wifi/infineon/airoc_wifi.c

```diff
@@ -760,6 +761,89 @@ static int airoc_mgmt_ap_disable(const struct device *dev)
 	return 0;
 }
 
+static int airoc_iface_status(const struct device *dev,
+                              struct wifi_iface_status *status)
+{
+        if (airoc_if == NULL) {
```

Comment:
I am seeing many coding style issues in this commit, wonder why compliance checker script does not complain about them. Perhaps try to run `clang-format` for you changes or update the code manually.
Lets start with this one, we typically introduce variables at the start of the function.